mmcv/runner/checkpoint.py

```diff
@@ -503,6 +504,8 @@ def load_checkpoint(model,
                     filename,
                     map_location=None,
                     strict=False,
+                    rmword='module',
```

Comment:
Change this to
```python
revise_key=(r'^module.', '')
```

`revise_key` accepts both a tuple of two regexes, or a list of tuples.

Reply:
emm maybe a dict is also acceptable.

```python
revise_key={r'^module.': ''}
```

Reply:
Hi, thanks for your suggestion! Modified. BTW, it seems that this one can cover the _load_checkpoint_with_prefix() function in the sense of functionality.  While the latter may be more user-friendly.